.github/cloudbuild/pos-pr-open-to-main.yaml

```diff
@@ -130,7 +132,12 @@ steps:
   args:
   - -c
   - |
-    gh pr comment $_PR_NUMBER --repo $_HEAD_REPO_URL --body "Access the Point-of-Sale at: [$(cat /workspace/pr-$_PR_NUMBER.ip)](http://$(cat /workspace/pr-$_PR_NUMBER.ip))"
+    cat <<EOF > /workspace/gh-comment.txt -
```

Comment:
question: I see that it's setup so that you're commenting on the the PRs - is there preference for this, or instead is there a way to comment as github bot instead? 
(no need to be fixed in this PR though!)

Reply:
This was merged as part of this PR: https://github.com/GoogleCloudPlatform/point-of-sale/pull/101. I've made some changes to it to comment the IPs of both deployments. 

The reason why I had to comment with my personal access token is because there is no way in GH to create a token specific to a repository or a team or an organization. Tokens are associated to user accounts. 

For sure we can use a bot,  however, we will have to implement a bot. I have backlogged that work in an follow up issue: https://github.com/GoogleCloudPlatform/point-of-sale/issues/102